src/Models/H5PContent.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace EscolaLms\HeadlessH5P\Models;
+
+use App\User;
+use Illuminate\Database\Eloquent\Model;
+use EscolaLms\HeadlessH5P\Models\H5PLibrary;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+//use App\Models\User;
+
+class H5PContent extends Model
+{
+    protected $table = 'hh5p_contents';
+
+    protected $primaryKey = 'id';
+    protected $fillable = [
```

Comment:
Isn't it easier to just use `$guarded = [ 'id' ]`
instead of listing every fillable field (which seems to be every field apart from `id`)